src/React.Sample.Mvc6/Startup.cs

```diff
@@ -59,9 +59,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 				// send the request to the following path or controller action.
 				app.UseErrorHandler("/Home/Error");
 			}
-
-			app.UseRequestServices();
```

Comment:
If this is no longer required, can you please update the documentation too? https://github.com/reactjs/React.NET/blob/master/site/jekyll/getting-started/aspnet5.md